### ISSUE TITLE
Ruby ~> 2.5 support

### ISSUE THREAD

**Issue by maestromac** [jakeonrails/fix-db-schema-conflicts#26]
Ruby 2.5.0 was recently released. Would be great if this gem supports it.

```
Dumping database schema with fix-db-schema-conflicts gem
Error: Unknown Ruby version 2.5 found in `.ruby-version`.
Supported versions: 2.1, 2.2, 2.3, 2.4
```

**Comment by ideasasylum** [comment]
@maestromac This is an error message from rubocop and can be resolved by updating rubocop to ~> 0.52.0 (later versions of rubocop introduce other errors). Though 0.52.0 does give the following deprecation warning:

```
Dumping database schema with fix-db-schema-conflicts gem
Parser::Source::Rewriter is deprecated.
Please update your code to use Parser::Source::TreeRewriter instead
```

but works fine in my testing

**Comment by maestromac** [comment]
@ideasasylum whoa that worked. Thank you very much!
